diff --git a/javascript/utils/index.js b/javascript/utils/index.js
index bc3c28fe128f9775b0fbcd2e56e5ffa0a3b4a15c..9d5271ee33b838dec56d9f72a929d7f9afa0b596 100644
--- a/javascript/utils/index.js
+++ b/javascript/utils/index.js
@@ -1,11 +1,10 @@
 import React from 'react';
 import {
-  ViewPropTypes,
-  View,
   NativeModules,
   findNodeHandle,
   Platform,
 } from 'react-native';
+import { ViewPropTypes } from 'deprecated-react-native-prop-types';
 import resolveAssetSource from 'react-native/Libraries/Image/resolveAssetSource';
 
 function getAndroidManagerInstance(module) {
@@ -20,7 +19,7 @@ function getIosManagerInstance(module) {
   return NativeModules[getIOSModuleName(module)];
 }
 
-export const viewPropTypes = ViewPropTypes || View.props;
+export const viewPropTypes = ViewPropTypes;
 
 export function isAndroid() {
   return Platform.OS === 'android';
