diff --git a/Libraries/Animated/nodes/AnimatedNode.js b/Libraries/Animated/nodes/AnimatedNode.js
index 29ca6798afc3932a795b149aacc29cfd7e75b510..aa5781d08fceab1d71ed49320312e8ab02b1c088 100644
--- a/Libraries/Animated/nodes/AnimatedNode.js
+++ b/Libraries/Animated/nodes/AnimatedNode.js
@@ -122,9 +122,11 @@ export default class AnimatedNode {
 
   __callListeners(value: number): void {
     const event = {value};
-    this._listeners.forEach(listener => {
-      listener(event);
-    });
+    if (Array.isArray(this._listeners)) {
+      this._listeners.forEach(listener => {
+        listener(event);
+      });
+    }
   }
 
   __getNativeTag(): number {
