diff --git a/Libraries/Animated/nodes/AnimatedNode.js b/Libraries/Animated/nodes/AnimatedNode.js
index 9cf3b990d18c48800724641e5a1253d598c71e5d..e60b24d3f025a03a02696e99689c363d1be06d99 100644
--- a/Libraries/Animated/nodes/AnimatedNode.js
+++ b/Libraries/Animated/nodes/AnimatedNode.js
@@ -122,9 +122,11 @@ export default class AnimatedNode {
 
   __callListeners(value: number): void {
     const event = {value};
-    this._listeners.forEach(listener => {
-      listener(event);
-    });
+    if (Array.isArray(this._listeners)) {
+      this._listeners.forEach(listener => {
+        listener(event);
+      });
+    }
   }
 
   __getNativeTag(): number {
