diff --git a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetHandle/BottomSheetHandleContainer.tsx b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetHandle/BottomSheetHandleContainer.tsx
index 0223843..823d541 100644
--- a/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetHandle/BottomSheetHandleContainer.tsx
+++ b/node_modules/@gorhom/bottom-sheet/src/components/bottomSheetHandle/BottomSheetHandleContainer.tsx
@@ -135,10 +135,11 @@ function BottomSheetHandleContainerComponent({
     [animatedLayoutState]
   );
   const handleBoundingClientRect = useCallback(
-    ({ height }: BoundingClientRect) => {
+    (params: BoundingClientRect) => {
+      if (!params?.height) return;
       animatedLayoutState.modify(state => {
         'worklet';
-        state.handleHeight = height;
+        state.handleHeight = params.height;
         return state;
       });

@@ -148,7 +149,7 @@ function BottomSheetHandleContainerComponent({
           method: 'handleBoundingClientRect',
           category: 'layout',
           params: {
-            height,
+            height: params.height,
           },
         });
       }
diff --git a/node_modules/@gorhom/bottom-sheet/src/hooks/useBoundingClientRect.ts b/node_modules/@gorhom/bottom-sheet/src/hooks/useBoundingClientRect.ts
index cc85c8c..9d55c19 100644
--- a/node_modules/@gorhom/bottom-sheet/src/hooks/useBoundingClientRect.ts
+++ b/node_modules/@gorhom/bottom-sheet/src/hooks/useBoundingClientRect.ts
@@ -56,7 +56,7 @@ export function useBoundingClientRect(
     }

     // @ts-ignore 👉 https://github.com/facebook/react/commit/53b1f69ba
-    if (ref.current.unstable_getBoundingClientRect !== null) {
+    if (ref.current.unstable_getBoundingClientRect === 'function') {
       // @ts-ignore https://github.com/facebook/react/commit/53b1f69ba
       const layout = ref.current.unstable_getBoundingClientRect();
       handler(layout);
@@ -64,9 +64,9 @@ export function useBoundingClientRect(
     }

     // @ts-ignore once it `unstable_getBoundingClientRect` gets stable 🤞.
-    if (ref.current.getBoundingClientRect !== null) {
+    if (ref.current.getBoundingClientRect === 'function') {
       // @ts-ignore once it `unstable_getBoundingClientRect` gets stable.
-      const layout = ref.current.getBoundingClientRect();
+      const layout = ref.current?.unstable_getBoundingClientRect?.();
       handler(layout);
     }
   });