diff --git a/node_modules/@react-native-mapbox-gl/maps/javascript/utils/index.js b/node_modules/@react-native-mapbox-gl/maps/javascript/utils/index.js
index bc3c28f..2218010 100644
--- a/node_modules/@react-native-mapbox-gl/maps/javascript/utils/index.js
+++ b/node_modules/@react-native-mapbox-gl/maps/javascript/utils/index.js
@@ -1,11 +1,10 @@
 import React from 'react';
 import {
-  ViewPropTypes,
-  View,
   NativeModules,
   findNodeHandle,
   Platform,
 } from 'react-native';
+import { ViewPropTypes } from 'deprecated-react-native-prop-types';
 import resolveAssetSource from 'react-native/Libraries/Image/resolveAssetSource';
 
 function getAndroidManagerInstance(module) {
@@ -20,7 +19,7 @@ function getIosManagerInstance(module) {
   return NativeModules[getIOSModuleName(module)];
 }
 
-export const viewPropTypes = ViewPropTypes || View.props;
+export const viewPropTypes = ViewPropTypes;
 
 export function isAndroid() {
   return Platform.OS === 'android';
