diff --git a/node_modules/react-native-calendar-events/android/src/main/java/com/calendarevents/RNCalendarEvents.java b/node_modules/react-native-calendar-events/android/src/main/java/com/calendarevents/RNCalendarEvents.java
index 782a0da..4b43154 100644
--- a/node_modules/react-native-calendar-events/android/src/main/java/com/calendarevents/RNCalendarEvents.java
+++ b/node_modules/react-native-calendar-events/android/src/main/java/com/calendarevents/RNCalendarEvents.java
@@ -543,7 +543,6 @@ public class RNCalendarEvents extends ReactContextBaseJavaModule implements Perm
 
             if (recurrenceRule.hasKey("frequency")) {
                 String frequency = recurrenceRule.getString("frequency");
-                String duration = "PT1H";
                 Integer interval = null;
                 Integer occurrence = null;
                 String endDate = null;
@@ -555,10 +554,6 @@ public class RNCalendarEvents extends ReactContextBaseJavaModule implements Perm
                     interval = recurrenceRule.getInt("interval");
                 }
 
-                if (recurrenceRule.hasKey("duration")) {
-                    duration = recurrenceRule.getString("duration");
-                }
-
                 if (recurrenceRule.hasKey("occurrence")) {
                     occurrence = recurrenceRule.getInt("occurrence");
                 }
@@ -589,9 +584,6 @@ public class RNCalendarEvents extends ReactContextBaseJavaModule implements Perm
                 }
 
                 String rule = createRecurrenceRule(frequency, interval, endDate, occurrence, daysOfWeek, weekStart, weekPositionInMonth);
-                if (duration != null) {
-                    eventValues.put(CalendarContract.Events.DURATION, duration);
-                }
                 if (rule != null) {
                     eventValues.put(CalendarContract.Events.RRULE, rule);
                 }
